precovery/precovery_db.py

```diff
@@ -51,6 +51,51 @@ class PrecoveryCandidate:
     distance_arcsec: float
     dataset_id: str
 
+    @classmethod
+    def from_observed_ephem(
+        cls, obs: Observation, frame: HealpixFrame, ephem: Ephemeris
+    ):
+        # Calculate the HEALpixel ID for the predicted ephemeris of
+        # the orbit with a high nside value (k=15, nside=2**15) The
+        # indexed observations are indexed to a much lower nside but
+        # we may decide in the future to re-index the database using
+        # different values for that parameter. As long as we return a
+        # Healpix ID generated with nside greater than the indexed
+        # database then we can always down-sample the ID to a lower
+        # nside value
+        healpix_id = int(radec_to_healpixel(ephem.ra, ephem.dec, nside=CANDIDATE_NSIDE))
+
+        return cls(
+            # Observation data:
+            observation_id=obs.id.decode(),
+            mjd=obs.mjd,
+            ra_deg=obs.ra,
+            dec_deg=obs.dec,
+            ra_sigma_arcsec=obs.ra_sigma / ARCSEC,
```

Comment:
I don't really know. They're both stored internally in degrees, but we present error estimates in arcseconds. This was introduced in https://github.com/B612-Asteroid-Institute/precovery/commit/4c603b3732e9630cce30d3f391ab0e1ef9d40400. Maybe @moeyensj can explain the reasoning.

Reply:
@spenczar @akoumjian This is purely a conversion to the most easily interpretable scale for orbital uncertainty mapped to observations. Actual on-sky positions remain in degrees while all uncertainties go to arcseconds because errors are roughly on that scale. So we set tolerances in arcseconds and report astrometric errors to the MPC in arcseconds. One fun tidbit is that if we wanted to calculate the chi^2 per coordinate with our current structure we would just do (residual_ra_arcsec)^2 / (ra_sigma_arcsec)^2 as well.